src/main/java/greencity/dto/discount/DiscountValueDto.java

```diff
@@ -4,19 +4,17 @@
 import greencity.dto.specification.SpecificationNameDto;
 import javax.validation.Valid;
 import lombok.AllArgsConstructor;
-import lombok.Builder;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import org.hibernate.validator.constraints.Range;
 
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
-@Builder
-public class DiscountDto {
-    @Range(min = ValidationConstants.DISCOUNT_VALUE_MIN,
-        max = ValidationConstants.DISCOUNT_VALUE_MAX)
-    private int value;
+public class DiscountValueDto {
+    @Range(min = 0, message = ValidationConstants.DISCOUNT_VALUE_MIN_ERROR)
+    @Range(max = 100, message = ValidationConstants.DISCOUNT_VALUE_MAX_ERROR)
+    private int value = 0;
```

Comment:
@Range has "max" and "min" value, so you can past here single annotation with one error message like "has to be more than 0 and less than 100"

Reply:
ok